src/mmw/apps/modeling/views.py

```diff
@@ -235,8 +237,17 @@ def choose_worker():
     def predicate(worker_name):
         return settings.STACK_COLOR in worker_name or 'debug' in worker_name
 
+    @retry(Exception, delay=0.5, backoff=2, tries=3)
```

Comment:
I still got an Exception by calling this while restarting the celeryd service. Consider increasing the number of tries. What do you think about making the maximum retry duration around 10-15 seconds?